src/components/tasks/card/index.tsx

```diff
@@ -66,6 +68,29 @@ const Card: FC<Props> = ({
   if (isTaskOverdue()) {
     cardClassNames.push(classNames.overdueTask);
   }
+
+  function renderDate(fromNowEndsOn: string, shouldEdit: boolean){
+    if(shouldEdit){
```

Comment:
Wouldn't it be better to create a ternary operator?
It seems like it is either this or that right?

Reply:
I did,
Isn't this method cleaner and simpler than that?
![image](https://user-images.githubusercontent.com/78958441/184409027-625f5685-e828-4351-aab0-ef8aa4d08bf3.png)

Reply:
Doesn't this snippet does the entire work of the function created?

Reply:
And if we wouldn't be utilising anywhere else, then having this snippet is better than a separate function call and returning the same.

If we had multiple functions, then it would be good to have a separate function like one created if the scope of card is limited then yes this sound right